KarhooSDK/Api/Response/MeetingPoint.swift

```diff
@@ -10,32 +10,38 @@ public struct MeetingPoint: Codable {
     public let position: Position
     public let instructions: String
     public let type: MeetingPointType
+    public let note: String
 
     public init(position: Position = Position(),
                 instructions: String = "",
-                type: MeetingPointType = .notSet) {
+                type: MeetingPointType = .notSet,
```

Comment:
Doesn't need to be done now, but in MeetingPointType, make sure that Airport nor train station is listed. Can be done in Part 2.